src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,36 @@
+import { FilterStatus } from '../types/FilterStatus';
+import { TodoFilter } from './TodoFilter';
+
+type Props = {
+  todosLeft: number;
+  isHasCompletedTodos: boolean;
```

Comment:
Mind the grammar (here and in the other places)
```suggestion
  hasCompletedTodos: boolean;
```